proto/sf/ethereum/type/v2/type.proto

```diff
@@ -928,92 +1019,159 @@ message GasChange {
 
   enum Reason {
     REASON_UNKNOWN = 0;
-    // REASON_CALL is the amount of gas that will be charged for a 'CALL' opcode executed by the EVM
+    
+    // REASON_CALL is the amount of gas that will be charged for a 'CALL' opcode executed by the EVM.
+    // This is a Firehose-specific reason not directly mapped to Geth tracing reasons.
```

Comment:
This is an opcode generated gas change reason from mapping at https://github.com/streamingfast/go-ethereum/blob/firehose-fh3.0/eth/tracers/firehose.go#L1247-L1264 and they are triggered from evm.Config.Tracer.OnGasChange(gasCopy, gasCopy-cost, tracing.GasChangeCallOpCode)

Please adjust all gas change reason containing now `This is a Firehose-specific reason not directly mapped to Geth tracing reasons.` with a more appropriate comment based on the new knowledge above that it's an opcode related gas change and not a firehose-specific reason.